recipes/meteor/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "meteor" %}
+{% set version = "2.0.9" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+
+source:
+  url: https://github.com/metagenopolis/meteor/archive/{{ version }}.tar.gz
+  sha256: e828010c5dd52e04156f6f0263d9d2635831d1253df9a5e0dc47becbd8e45c43
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  run_exports:
+    - {{ pin_subpackage('meteor', max_pin="x.x") }}
+
+requirements:
+    host:
+        - python =3.10.13
```

```suggestion
        - python >=3.10
```
It requires 3.10+ as in here: https://github.com/metagenopolis/meteor?tab=readme-ov-file#dependencies